Resources/Textures/_Harmony/Clothing/Head/Hardsuits/qmsuit.rsi/meta.json

```
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprites by rot_witch (Discord)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "icon-flash"
+    },
+    {
+      "name": "equipped-head",
+      "directions": 4
+    },
+    {
+      "name": "equipped-head-light",
+      "directions": 4
+    },
+    {
+      "name": "equipped-head-unshaded",
+      "directions": 4
+    },
+    {
+      "name": "equipped-head-vox",
+      "directions": 4
+    },
+    {
+      "name": "equipped-head-light-vox",
+      "directions": 4
+    },
+    {
+      "name": "equipped-head-unshaded-vox",
+      "directions": 4
+    }
+  ]
+}
```

```suggestion
{
    "version": 1,
    "license": "CC-BY-SA-3.0",
    "copyright": "Made by rot_witch (Discord)",
    "size": {
        "x": 32,
        "y": 32
    },
    "states": [
        {
            "name": "icon"
        },
        {
            "name": "icon-flash"
        },
        {
            "name": "equipped-head",
            "directions": 4
        },
        {
            "name": "equipped-head-light",
            "directions": 4
        },
        {
            "name": "equipped-head-unshaded",
            "directions": 4
        },
        {
            "name": "equipped-head-vox",
            "directions": 4
        },
        {
            "name": "equipped-head-light-vox",
            "directions": 4
        },
        {
            "name": "equipped-head-unshaded-vox",
            "directions": 4
        }
    ]
}
```

Indentation, and also you're missing EOF (You need an empty line at the end of your files)